docs/source/installation/for_dev.rst

```diff
@@ -146,4 +146,4 @@ For Developers with pip
    python3 -m pip install .
 
 Please see :doc:`../contributing` for more details about contributing to Manim.
-Since `pip` doesn't implement editable installations from our pyproject.toml
+Since `pip` doesn't implement editable installations from our ``pyproject.toml``
```

Comment:
though this wouldn't be the case always. https://github.com/python/peps/pull/1944